qa/tasks/cephfs/test_volumes.py

```diff
@@ -7888,3 +7888,29 @@ def test_unparseable_metafile_on_legacy_to_v1_upgrade(self):
 
         # remove group
         self._fs_cmd("subvolumegroup", "rm", self.volname, group)
+
+class TestPerModuleFinsherThread(TestVolumesHelper):
+    """
+    Per module finisher thread tests related to mgr/volume cmds.
+    This is used in correspondence with check_counter wiht min val being 4
```

Comment:
Maybe rather than correspondence you mean in parallel, in conjunction, or something similar (this is also just a nit pick)

Reply:
Done